UnicornRender/video/include/unicorn/video/PerspectiveCamera.hpp

```diff
@@ -23,7 +23,7 @@ class PerspectiveCamera final : public CameraProjection
     UNICORN_EXPORT PerspectiveCamera(system::Window* window, glm::mat4& cameraProj);
 
     /** @brief Changes fov */
-    UNICORN_EXPORT void Scroll(float yoffset);
+    UNICORN_EXPORT void Zoom(float zoomOffset);
```

Comment:
How is zoomOffset connected to fov? How shall user understand how much FOV will change with this offset?

Reply:
Fov originally is 45. User change it by adding zoomOffset value. I don't think it have measure like meters. Just value. Will extend documentation here